### PR TITLE
03 basic sql

### PR DESCRIPTION
**Pull request by v-smatveyenka**

### CODE REVIEW

File: 03_Basic SQL/Scripts/1.1.1_Orders_after_6.5.98.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿SELECT OrderID, ShippedDate, ShipVia 
+  FROM [dbo].[Orders]
+  WHERE ShippedDate>=convert(datetime,'05.06.1998', 121) AND ShipVia>=2
```
Comment (ikolyaskin) [inline 03_Basic SQL/Scripts/1.1.1_Orders_after_6.5.98.sql]: I think with SQL Server there is no need to convert string to datetime, as you can just use dates in queries in this format: `'1998-12-31'`. They will also work with `>`, `<`, `=`

File: 03_Basic SQL/Upgrate 1.0 - 1.1.sql [hunk 2]
```diff
@@ -0,0 +1,13 @@
+IF OBJECT_ID(N'[Northwind].[CreditCards]') IS NULL
```
Comment (ikolyaskin) [inline 03_Basic SQL/Upgrate 1.0 - 1.1.sql]: I think you wanted to call these files "Update" :)

**Review by ikolyaskin — APPROVED** [review]

File: 03_Basic SQL/Upgrate 1.0 - 1.1.sql (hunk 2 above)
Comment (v-smatveyenka) [reply 03_Basic SQL/Upgrate 1.0 - 1.1.sql]: yes, thank you)

File: 03_Basic SQL/Scripts/1.1.1_Orders_after_6.5.98.sql (hunk 1 above)
Comment (v-smatveyenka) [reply 03_Basic SQL/Scripts/1.1.1_Orders_after_6.5.98.sql]: fixed

**Pull request merged**